starknet-foundry/crates/cast/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "cast"
 description = "Cast - a Starknet Foundry CLI client"
-version = "0.1.0"
-edition = "2021"
+version.workspace = true
```

Comment:
I've inherited the workspace settings for `cast` and `forge`, not sure if it makes sense to do the same for cheatable starknet and test collector. It kind of make sense to have separate versioning in them.